tycho-core/src/main/java/org/eclipse/tycho/p2resolver/TargetPlatformFactoryImpl.java

```diff
@@ -291,6 +296,31 @@ public P2TargetPlatform createTargetPlatform(TargetPlatformConfigurationStub tpC
         return targetPlatform;
     }
 
+    private void gatherP2InfUnits(ReactorProject reactorProject, Set<IInstallableUnit> externalUIs) {
+        if (reactorProject == null) {
+            return;
+        }
+        AdviceFileAdvice advice;
+        if (PackagingType.TYPE_ECLIPSE_PLUGIN.equals(reactorProject.getPackaging())) {
+            advice = new AdviceFileAdvice(reactorProject.getArtifactId(), Version.parseVersion("1.0.0"),
```

Comment:
Is this hardcoded version `1.0.0` irrelevant? what if the declared unit sets its version to anything else?
(same below on the `else` branch of the `if`)

Reply:
The version is only used to check if the advice should be applied to a unit, as we are not interested in the advice itself but only the additional units it defines the version has actually no influence here.